server/models/PetType.js

```diff
@@ -23,6 +23,7 @@ class PetType {
       throw (error)
     }
   }
+

```

Comment:
We can actually remove this empty line.  When a file is ending it's ok that there are no lines between a chain of closing brackets